src/app/src/main/kotlin/org/vaccineimpact/api/app/serialization/FlexibleDataTable.kt

```diff
@@ -0,0 +1,108 @@
+package org.vaccineimpact.api.app.serialization
```

Comment:
There is quite a lot of code here that is shared with `DataTable`. I think it would be nicer to extend that class and just override the necessary bits? e.g. Both classes construct and flush the writers identically.